security/jwt/src/main/java/io/helidon/security/jwt/Jwt.java

```diff
@@ -284,7 +290,7 @@ private Jwt(Builder builder) {
         this.headers = builder.headerBuilder.build();
 
         // known payload
-        this.issuer = builder.issuer;
+        this.issuers = builder.issuers;
```

Comment:
This should be `List.copyOf(builder.issuers)` to have a defensive copy and to have it immutable (as we return the list in API)